pkg/objectcache/applicationprofilecache/applicationprofilecache.go

```diff
@@ -216,6 +216,7 @@ func (ap *ApplicationProfileCacheImpl) addApplicationProfile(_ context.Context,
 			return
 		}
 		ap.slugToAppProfile.Set(apName, fullAP)
+		logger.L().Debug("added pod to application profile cache", helpers.String("name", apName))
```

Comment:
```suggestion
		logger.L().Info("added pod to application profile cache", helpers.String("name", apName))
```

Reply:
I think you advised to change this to debug like in the network neighbors cache...